media/admin/com_virtuemart/models/zasilkovna_src/VirtueMartModelZasilkovna/Order/Detail.php

```diff
@@ -88,8 +107,25 @@ public function validateFormData(array $formData)
                 );
             }
         }
+    }
 
-        return $validationReport;
+    private function validateMinimumValues(array $formData, array $fields, DetailFormValidationReport $validationReport)
+    {
+        $hasAddedGeneralError = false;
```

Comment:
Pokud tam budou chyby z obou metod, nezobrazí se "general error" dvakrát?

Reply:
Nezobrazí, protože si Joomla filtruje duplikátní chybové hlášky v metodě `enqueueMessage()`.

Reply:
Čistší by bylo rozdělit validaci a volání `addError` - tedy nejprve zvalidovat a vrátit pole chyb, a pokud jsou chyby, vypsat "general error" a všechny chyby.

Reply:
Mas pravdu, vypada to mnohem lepe. Posilam upravu.